src/util/vocabularyChecker/detectWrongSyntax.ts

```diff
@@ -44,18 +54,18 @@ export function getPositionsOfInvalidWord(invalidWord: string, text: string): Po
 }
 
 /**
- * Bundling the functionalities of @function collectAllInvalidWords and
- * @function removeDuplicates.
+ * Bundling the functionalities of @function collectAllInvalidWords and @function removeDuplicates.
```

Comment:
The @function keyword is not supposed to be used like this. Rather it serves as an indicator that an object, wie might not appear to be a function, is actually a function:
https://devdocs.io/jsdoc/tags-function
An example could be `const iLookLikeAVariable = () => { // Do Something};`